sample/src/main/java/com/daasuu/exoplayerfilter/MainActivity.java

```diff
@@ -138,7 +138,7 @@ private void setUpSimpleExoPlayer() {
 
     private void setUoGlPlayerView() {
         ePlayerView = new EPlayerView(this);
-        ePlayerView.setSimpleExoPlayer(player);
+        ePlayerView.setExoPlayer(player);
```

Comment:
Actually, there are a lot of things we added after forking, since Luke had been working on this repository. Should I look into the commits and write the changes? or I can start writing change logs starting from my change list.

Reply:
Yeah, please start a changelog with Luke's changed.

With the package id i mean: update to `com.github.Innovattic:ExoPlayerFilter:0.4.2`

Reply:
THe readme still says "This library depends ExoPlayer core 2.10.2" and there's a few more references to 2.10.2.

Reply:
Fixed